docs/stdlib/numbers.rst

```diff
@@ -251,14 +263,14 @@ from :eql:type:`str` and :eql:type:`json`.
 
     .. note::
 
-        Caution is advised when casting ``bigint`` values into
-        ``json``. The JSON specification does not have a limit on
-        significant digits, so a ``bigint`` number can be losslessly
+        Caution is advised when casting :eql:type:`bigint` values into
+        :eql:type:`json`. The JSON specification does not have a limit on
+        significant digits, so a big integer number can be losslessly
         represented in JSON. However, JSON decoders in many languages
-        will read all such numbers as some kind of 32- or 64-bit
+        will read all such numbers as some kind of 32 or 64-bit
```

Comment:
```suggestion
        will read all such numbers as some kind of 32-bit or 64-bit
```

I'm with you in not being crazy about the dangling hyphen, but at the same time, this change disconnects "32" from "bit." Let's get around it by being a little more verbose.